src/Microsoft.PowerShell.Commands.Management/commands/management/Service.cs

```diff
@@ -1106,14 +1098,9 @@ private bool HaveAllDependentServicesStopped(ICollection<ServiceController> depe
         /// <param name="services">A list of services.</param>
         internal void RemoveNotStoppedServices(List<ServiceController> services)
         {
-            foreach (ServiceController service in services)
-            {
-                if (service.Status != ServiceControllerStatus.Stopped &&
-                    service.Status != ServiceControllerStatus.StopPending)
-                {
-                    services.Remove(service);
-                }
-            }
+            services.RemoveAll(service =>
```

Comment:
Please add a comment why we have to use LINQ.

Reply:
This is not LINQ. It's a method of `List<T>`.

Reply:
It is just better. Please add a comment.